admin/tool/policy/classes/api.php

```diff
@@ -162,6 +162,28 @@ public static function get_policy_version_status($policy, $versionid) {
         return $status;
     }
 
+    /**
+     * Returns a list of all current policy versions id.
+     *
+     * @return array list of policy version ids.
+     */
+    public static function get_current_policy_versionids() {
+        global $DB;
+
+        $sql = "SELECT d.currentversionid
+                  FROM {tool_policy} d
+                 WHERE d.currentversionid is NOT NULL";
+        $params = [];
+
+        $result = $DB->get_records_sql($sql, $params);
+        $versionids = [];
+        foreach ($result as $row) {
+            $versionids[] = $row->currentversionid;
```

Comment:
If there is really need for a method like this, would not it be better to also keep the policy id? So that the returned structure would look like [$policyid] => [$currentversionid] ? But without seeing this actually used, it is hard to guess.